modules/nodes-adding-bare-metal-host-to-cluster-using-cli.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-cli_{context}"]
+= Adding a bare metal host to the cluster and provisioning a bare metal node using the CLI
+
+You can add bare metal hosts to the cluster and then provision them as bare metal nodes using the CLI.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a bare metal host for use in the cluster by completing the following actions:
+.. Create the following `BareMetalHost` Custom Resource (CR) that defines the `Secret` credentials and the required cluster node namespace, for example `openshift-worker`. Save the YAML in the `openshift-worker-0-3-namespace.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-0-3-bmc-secret
+type: Opaque
+data:
+  username: <secret-username>
+  password: <secret-password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-0-3
+spec:
+  online: true
+  bmc:
+    address: <BMC address>
+    credentialsName: openshift-worker-0-3-bmc-secret
+    disableCertificateVerification: True
+    username: <username>
+    password: <password>
+  bootMACAddress: <host boot MAC address>
+  hardwareProfile: unknown
+----
+.. Create the namespace by running the following command:
```

Comment:
In a bare metal cluster I believe (?) the `openshift-machine-api` namespace will already exist. Is it accurate to say this will create the namespace? Or does this command just apply the YAML and put both CRs into that existing namespace?

Reply:
Yup - you are correct. will amend.